content/en/security/cloud_security_management/troubleshooting/vulnerabilities.md

```diff
@@ -47,6 +47,7 @@ ERROR | (pkg/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go:80
 The workaround for this issue is to set the configuration option:
 - For containerd: set `discard_unpacked_layers=false` in the containerd configuration file.
 - For Helm: set `datadog.sbom.containerImage.uncompressedLayersSupport: true` in your `values.yaml` file.
+- For Datadog Operator: set `features.sbom.containerImage.uncompressedLayersSupport: true` in your `datadog-agent.yaml` file.
```

Comment:
For Datadog Operator: set `features.sbom.containerImage.uncompressedLayersSupport` at `true` in your DatadogAgent CRD.

Reply:
Fixed